src/components/Searchbar.js

```diff
@@ -34,7 +34,7 @@ function saveLocalSearch() {
     if (dateTo) {
         dateTo = new Date(dateTo);
     } else {
-        dateTo = date;
+        dateTo = dateFrom;
```

Comment:
why would u set up dateTo and dateFrom to be equal to each other? 


Can we use a ternary operator
it can look something like this:
`dateTo == null ? new Date(dateTo) : dateFrom`